sample-plugin/pom.xml

```diff
@@ -9,11 +9,10 @@
     </parent>
     <groupId>io.jenkins.tools.bom</groupId>
     <artifactId>sample</artifactId>
-    <version>${revision}${changelist}</version>
+    <version>${changelist}</version>
     <packaging>hpi</packaging>
     <properties>
-        <revision>30</revision>
-        <changelist>-SNAPSHOT</changelist>
+        <changelist>999999-SNAPSHOT</changelist>
```

Comment:
Still duplicated here, but overridden when built in release mode with `-Dset.changelist`, and should no longer ever need to be edited in sources.